<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Segmentation with SAM </string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>File :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>391</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="openFile">
   <property name="geometry">
    <rect>
     <x>494</x>
     <y>10</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>50</y>
     <width>331</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Classes</string>
   </property>
   <widget class="QListWidget" name="classWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>101</y>
      <width>311</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Class Name :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="classEdit">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>60</y>
      <width>241</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addClass">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeClass">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>60</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="maskLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>191</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>Class</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>330</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Start </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
